src/test/java/org/jenkinsci/plugins/workflow/steps/scm/SCMStepTest.java

```diff
@@ -45,118 +45,108 @@
 import static org.junit.Assert.*;
 import org.junit.ClassRule;
 import org.junit.Rule;
-import org.junit.runners.model.Statement;
 import org.jvnet.hudson.test.BuildWatcher;
 import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.RestartableJenkinsRule;
 
 public class SCMStepTest {
 
     @ClassRule public static BuildWatcher buildWatcher = new BuildWatcher();
-    @Rule public RestartableJenkinsRule r = new RestartableJenkinsRule();
+    @Rule public RestartableJenkinsRule rr = new RestartableJenkinsRule();
     @Rule public GitSampleRepoRule sampleGitRepo = new GitSampleRepoRule();
     @Rule public SubversionSampleRepoRule sampleSvnRepo = new SubversionSampleRepoRule();
 
     @Issue("JENKINS-26100")
     @Test
     public void scmVars() throws Exception {
-        r.addStep(new Statement() {
-            @Override public void evaluate() throws Throwable {
-                sampleSvnRepo.init();
```

Comment:
[for review](https://github.com/jenkinsci/workflow-scm-step-plugin/pull/19/files?w=1#diff-862fb161c96b2514ed6613245a629e71L63)